corehq/form_processor/sql_functions/deprecate_form.sql

```diff
@@ -0,0 +1,29 @@
+DROP FUNCTION IF EXISTS deprecate_form(TEXT, TEXT, TIMESTAMP);
+
+CREATE FUNCTION deprecate_form(new_form_id TEXT, orig_id TEXT, edited_on TIMESTAMP) RETURNS VOID AS $$
```

Comment:
Yea, I initially had some issues with this but I think it was because I was using the `SQL` language. Will go through and do a clean up sometime.